src/gui/dialog/ExportDialog.h

```diff
@@ -26,7 +29,7 @@ class ExportDialog: public GladeGui {
     virtual void show(GtkWindow* parent);
     void initPages(int current, int count);
     bool isConfirmed() const;
-    PageRangeVector getRange();
+    std::vector<PageRangeEntry> getRange();
```

Comment:
```suggestion
    std::vector<PageRangeEntry> getRanges();
```

Reply:
I'm going to wait for your response in the other similar review comment.